src/ecpair.js

```diff
@@ -84,6 +84,17 @@ ECPair.fromWIF = function (string, network) {
   })
 }
 
+ECPair.fromUncheckedHex = function (hex, network, compressed) {
+	
+	// assume a network object (or default to bitcoin)
+	network = network || NETWORKS.bitcoin
+	
+	return new ECPair(BigInteger.fromHex(hex), null, {
+		compressed: compressed,
+		network: network
+	})
```

Comment:
Uses tabs, should use spaces instead

Reply:
I've updated the pull request to use spaces instead of tabs.